libraries/helpers.rb

```diff
@@ -0,0 +1,38 @@
+module Chocolatey
+  module Helpers
+    # Get the ChocolateyInstall directory from the environment.
+    def chocolatey_install
```

Comment:
Ok now that I look at the former function again it makes no sense because I don't understand why `ENV` would return multiple keys even if they are mixed case. However, the os should be doing the fallback for us in both cases. If both user and machine `ChocolateyInstall` env vars are set, windows should return the user and fall back to the machine. I remember you mentioning to me offline about trouble with the environment variables. Did you have issues prompting you to force the fallback?